todos-app/src/DataStructure.ts

```diff
@@ -0,0 +1,9 @@
+export interface ITodo {
+  id: string;
+  text: string;
+  completed: boolean;
+}
+
+export type TodoListType = {
```

Comment:
Don't seem like there is a reason to use both types and interfaces, let's make this consistent and use either types or interfaces

Reply:
don't need Type in the name

Reply:
At rocketparts, most engineers use types, just for your information, this way you also don't need to add I to your type names

Reply:
Fixed it. I decided to use types instead of interfaces